### PR TITLE
Return strin from get_lqip_src instead of class

### PR DESCRIPTION
**Pull request by ostowe**
Otherwise we can't know if it failed to retrieve the lqip src and will just end up using the full size image...which totally defeats the purpose :/

### CODE REVIEW

File: inc/components/class-image.php [hunk 1]
```diff
@@ -257,7 +257,9 @@ public function configure( $picture ) {
 	 * @return string
 	 */
 	public function get_aspect_ratio( $size_config ) {
-		$aspect_ratio = $size_config['aspect_ratio'] ?? $this->config['aspect_ratio'] ?? false;
+		$aspect_ratio = isset( $size_config['aspect_ratio'] ) ?
```
Comment (GFargo) [inline inc/components/class-image.php]: This should work... if you wanted to shorten, not sure it's 'simplification' ;)

```suggestion
		$aspect_ratio = ( $size_config['aspect_ratio'] ?? $this->config['aspect_ratio'] ) ?? false;
```

File: inc/components/class-image.php [hunk 2]
```diff
@@ -257,7 +257,9 @@ public function configure( $picture ) {
 	 * @return string
 	 */
 	public function get_aspect_ratio( $size_config ) {
```
Comment (GFargo) [inline inc/components/class-image.php]: Could add some nullable typehinting to the return value here
```suggestion
	public function get_aspect_ratio( $size_config ): ?string  {
```

File: inc/components/class-image.php [hunk 3]
```diff
@@ -377,13 +379,13 @@ public function get_lqip_src() {
 
 		// Return early if no aspect ratio is set.
 		if ( empty( $aspect_ratio ) ) {
-			return $this;
+			return '';
```
Comment (GFargo) [inline inc/components/class-image.php]: If you type-hinted this function, per comment above you'd want to update this to be `null`
```suggestion
			return null;
```

File: inc/components/class-image.php (hunk 2 above)
Comment (ostowe) [reply inc/components/class-image.php]: oo didn't know about these, nice!

**Pull request merged**